### PR TITLE
Tightened up SearchGames page

### PR DESCRIPTION
**Pull request by NickLiggett**
## What does this PR do?
- [] New feature (adds functionality)
- [x] Debugging/Refactor
 
### Where should your reviewer start?
- `src/components/SearchGames.js`
### What (if any) features are you implementing?
- Added a Search Button and cleaned up the search functionality.
### What (if anything) did you refactor?
- Ended up using a new dropdown menu.
### What issues does this PR close?
- 
### Is there anything that you need from your teammates?
- Try and break it!
### Any other comments, questions, or concerns?
-
### How Has This Been Tested?
- [x] I tested my changes in the browser
- [] I tested with Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by aspitz1 — APPROVED** [review]
Everything seems good to me! Thanks Nick.

**Comment by aspitz1** [comment]
I tried to break it, and it seems to be working well!

File: mock-data/MockGamesList.js [hunk 1]
```diff
@@ -41,35 +41,35 @@ const userGames = [
...
-    genres: ["First-Person Shooter"],
+    genres: ["Fantasy", "RPG"],
     consoles: ["Xbox", "PS5", "PC"],
   },
   {
     title: "Grand Theft Auto V",
     image:
       "https://upload.wikimedia.org/wikipedia/en/a/a5/Grand_Theft_Auto_V.png",
-    genres: ["Second-Person Shooter"],
+    genres: ["Person-Shooter"],
```
Comment (tfettig22) [inline mock-data/MockGamesList.js:51]: nice genre tag for GTA, made me lol

**Review by tfettig22 — APPROVED** [review]

**Review by tfettig22 — APPROVED** [review]
Nice job cleaning this up, I tried to break and cannot. seems like everything is in great shape!

**Comment by aspitz1** [comment]
Oh! I am just remembering that the Name in the search for a game screen needs to be a required field, I think. We may want to follow up with the backend on that.

**Pull request merged**